src/libraries/System.Runtime.Loader/tests/SatelliteAssemblies.cs

```diff
@@ -207,6 +207,7 @@ public void SatelliteLoadsCorrectly_FromName(string alc, string assemblyName, st
         }
         
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotInvariantGlobalization))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/41997", TestRuntimes.Mono)]
```

Comment:
IIRC this didn't fail on Browser, should we disable only on ~TestPlatforms.Browser?